### ISSUE TITLE
Difference between SSG tar.bz and .zip content

### ISSUE THREAD

**Issue by jmnielsen7** [OpenSCAP/scap-security-guide#2125]
I've noticed that if I download the tar.bz for SSG Release 1.33 that there is an entirely different hierarchy and content than if I download the zip file. The zip (scap-security-guide-0.1.33.zip) has the xccdf profiles that I need so that's the only one I ever grab. 

But since 1.34 was released today I cannot find the .zip with the profiles in it (1.33 released 5 files, but 1.34 currently only has 4 available <a href="https://github.com/OpenSCAP/scap-security-guide/releases/tag/v0.1.34">on this page</a> - missing the non-source zip), and I can't find the equivalent files I need in the tar.bz, but perhaps I just need a lesson on what the difference is in the content and the purpose of each.

-JN

**Comment by jmnielsen7** [comment]
Maybe I was impatient, since this was only released in the last hour? The fifth file just appeared in the last five minutes. Hmm... I still would like a clarification on why the contents of the tar and zip are different though.

**Comment by yuumasato** [comment]
Hello,

I uploaded the zip files some time ago.

The zip file contains DataStreams and kickstarts ready for use and the tarball contains the source code to build everything, it is mostly useful for packagers.

**Comment by jmnielsen7** [comment]
Huh, well maybe it was just the link missing in the "Downloads" section on the URL I linked to above, even though the file was there? I should have taken a screenshot but there was definitely only four links (the scap-security-guide-0.1.34.zip was missing there) showing in the download section when I posted this, because I was comparing it to the 1.33 release page in another tab and realized there was file missing on the 1.34 release page under the Downloads section. But in any case it's all good now.

And thanks for the explanation!
